### PR TITLE
doc: Add manual test for volume backup and restore by AWS IAM role

### PR DESCRIPTION
**Pull request by jenting**
https://github.com/longhorn/longhorn/issues/1526

### CODE REVIEW

File: docs/content/manual/release-specific/v1.1.1/aws-iam-role-for-volume-backup-restore.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    -n longhorn-system
+```
+10. On the Longhorn UI, click Settings. In the Backup section, set Backup Target to:
+```shell
+s3://<your-bucket-name>@<your-aws-region>/
+```
+11. In the Backup section, set Backup Target Credential Secret to:
+```shell
+aws-secret
+```
```
Comment (khushboo-rancher) [inline docs/content/manual/release-specific/v1.1.1/aws-iam-role-for-volume-backup-restore.md:149]: 12. Verify that the user is able to create/list/delete backups.

**Comment by khushboo-rancher** [comment]
1. Are we not going ahead with `AWS_IAM_ARN` solution?
2. For google backup store also, is it applicable?
3. I'm assuming the doc https://longhorn.io/docs/1.1.0/snapshots-and-backups/backup-and-restore/set-backup-target/ will be updated this.

**Comment by jenting** [comment]
> 1. Are we not going ahead with `AWS_IAM_ARN` solution?

Thank u, I have updated all PRs to use `AWS_IAM_ROLE_ARN`.

> 2. For google backup store also, is it applicable?

I'm not 100% sure, but I guess it no.
Since the solution needs AWS STS API to provide a temporary credential to the IAM role, we have to check if the GCP supports the same kind of service.

> 3. I'm assuming the doc https://longhorn.io/docs/1.1.0/snapshots-and-backups/backup-and-restore/set-backup-target/ will be updated with this.

Yes, the PR is at https://github.com/longhorn/website/pull/271

**Review by khushboo-rancher — APPROVED** [review]
LGTM

**Pull request merged**